uint8 status
uint8 RESET = 0
uint8 START = 1
---
bool is_success